assets/data/index-sections.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "title": "About Us",
+        "desc": "ICS Student Council was founded by students to better serve the ICS student body. We work to improve the lives of the students professionally, socially, and academically and deepen the connections students have with the Donald Bren School of Information and Computer Sciences and UCI Alumni."
+    },
+
+    {
+        "title": "Event Calendar",
+        "desc": "Check out our upcoming events!"
+    },
+
+    {
+        "title":"Thanks to Our Sponsors",
+        "desc": ""
+    }
+]
```

Comment:
Since this is page content and not  really dynamic data, I'd personally leave it in `index.js` and not have a json for it.

Reply:
Updated! index-sections.json is removed since the data is just page content.